content/projects/ecft-vs-pipeline.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ECFT VS-Pipeline
+date: 2023-04-15
+weight: 10
+nav: false
+# Uncomment sections below to enable them in your project
+#external_resources:
+#- name: ECFT-VS-pipeline Repository
+#  link: https://github.com/volkamerlab/ECFT-VS-pipeline
+#  icon: github
+people:
+- key: hamza.ibrahim
+- key: michael.backenkoehler 
+- key: andrea.volkamer
+cooperation:
+- name: Helmholtz-Institute for Pharmaceutical Research Saarland, HIPS
+  link: https://www.helmholtz-hips.de/en/
+---
+
+Finding the optimal docking pipeline for a target is challenging. Our project aims to develop a data-driven pipeline to find new hits for Energy coupling factor (ECF) transporters through the optimization of various structure-based virtual screening workflows.
+
+<!--more-->
+
+Energy coupling factor (ECF) transporters are novel, promising antimicrobial targets which mediate micronutrient transport into the cell. Therefore this protein is a promising drug target. Absence of ligands with a co-crystallized protein with ligand and its cryptic binding pocket make it a challenging target. However, previous computational work at the Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) revealed an opening pocket in ECF transporters for pantothenate.
```

Comment:
```suggestion
Energy-coupling factor transporters (ECFTs) are novel, promising antimicrobial targets which mediate micronutrient transport into the cell. Absence of co-crystallized protein-ligand complexes and its cryptic binding pocket make ECFT a challenging target. However, previous computational work at the Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) and collaboration partners revealed an opening pocket in ECFT for pantothenate, which is used as basis for this study.
```

Reply:
@hamzaibrahim21 add a link to the study

Reply:
@AndreaVolkamer Thanks for the feedback. I've added the link to the study as a reference. Please let me know if I should add it in another way, e.g. add to _publications_ section.